### PR TITLE
Added the daos_agent log to the troubleshooting sections (#2030)

### PR DESCRIPTION
**Pull request by mjean308**
Skip-build: true
Skip-test: true

Signed-off-by: Maureen Jean <maureen.jean@intel.com>

### CODE REVIEW

File: doc/admin/troubleshooting.md [hunk 1]
```diff
@@ -36,6 +36,7 @@ server operations:
 |Control Plane|control_log_file|/tmp/daos_control.log|
 |Data Plane|log_file|/tmp/daos_server.log|
 |[Privileged Helper](https://daos-stack.github.io/admin/deployment/#elevated-privileges)|helper_log_file|/tmp/daos_admin.log|
+|agent|log_file|/tmp/daos_agent.log|
```
Comment (mjmac) [inline doc/admin/troubleshooting.md]: This list of config parameters is for the daos_server.yml file, so it's a little confusing to add the agent-side parameter here.

Comment (mjean308) [reply doc/admin/troubleshooting.md]: Should it be more generic?  The section is on Log files and there is a daos_agent logfile.    Or maybe add a column for the configuration file that defines the log file?

Comment (mjmac) [reply doc/admin/troubleshooting.md]: Oh, I see this is for release/0.9. Well, what I was pointing out is that immediately before this table, there is the text: "On the server side, there are three log files created as part of normal server operations:" ... So now we've got the agent log mixed in with the server logs.

Comment (mjean308) [reply doc/admin/troubleshooting.md]: I will remove the daos_agent.log from the table

**Pull request merged**